plugins/base/base-test-utils/src/main/kotlin/renderers/JsoupUtils.kt

```diff
@@ -13,6 +16,7 @@ fun Element.match(vararg matchers: Any): Unit =
 
 open class Tag(val name: String, vararg val matchers: Any)
 class Div(vararg matchers: Any) : Tag("div", *matchers)
+class CodeTag(vararg matchers: Any) : Tag("code", *matchers)
```

Comment:
I'd name it `Code` as other tags are named exactly the same as their html equivalents